Packs/EWS/Integrations/SecurityAndCompliance/SecurityAndCompliance.yml

```diff
@@ -19,8 +19,9 @@ configuration:
   required: false
   type: 8
 description: This integration allows you to manage and interact with Microsoft security and compliance content search.
-display: O365 - Security And Compliance - Content Search
+display: O365 - Security And Compliance - Content Search (Deprecated)
 name: SecurityAndCompliance
+deprecated: true
```

Comment:
lets not deprecate yet, because if it is still used in news on-prem, it is still valid.